src/hacking/building-for-openharmony.md

```diff
@@ -4,189 +4,29 @@
-It supports Windows and MacOS.
-You can manage installed OpenHarmony SDKs by clicking File->Settings and selecting "OpenHarmony SDK".
-After setting a suitable installation path, you can select the components you want to install for each available API version.
-DevEco Studio will automatically download and install the components for you.
-
-
-#### Manual installation of the OpenHarmony SDK (e.g. on Linux)
-
-<div class="warning _note">
-    Before rushing and downloading the OH SDK from gitee as described here, please note that you will also need hvigor to compile applications.
-    hvigor is currently recommended to be downloaded via the HarmonyOS NEXT commandline tools package, which also contains a copy of the OpenHarmony SDK.
-</div>
-
-1. Go to the [OpenHarmony release notes] and select the version you want to compile for.
+## Step 1
+### Prerequisites
+1. Go to the [OpenHarmony release notes](https://gitee.com/openharmony/docs/blob/master/zh-cn/release-notes/Readme.md) and select the version you want to compile for.
 2. Scroll down to the section "Acquiring Source Code from Mirrors" and click the download link for the version of "Public SDK package for the standard system" matching your host system.
 3. Extract the archive to a suitable location.
 4. Switch into the SDK folder with `cd <sdk_folder>/<your_operating_system>`.
 5. Create a sub-folder with the same name as the API version (e.g 12 for SDK v5.0) and switch into it.
 6. Unzip the zip files of the individual components into the folder created in the previous step. Preferably use the `unzip` command on the command-line, or manually ensure that the unzipped bundles are called e.g. `native` and not `native-linux-x64-5.x.y.z`.
 
-The following snippet can be used as a reference for steps 4-6:
-```commandline
-cd ~/ohos-sdk/linux
-for COMPONENT in "native toolchains ets js previewer" do
-    echo "Extracting component ${COMPONENT}"
-    unzip ${COMPONENT}-*.zip
-    API_VERSION=$(cat ${COMPONENT}/oh-uni-package.json | jq -r '.apiVersion')
-    mkdir -p ${API_VERSION}
-    mv ${COMPONENT} "${API_VERSION}/"
-done
-```
-On windows, it is recommended to use 7zip to unzip the archives, since the windows explorer unzip tool is extremely slow.
-
-[DevEco Studio]: https://developer.huawei.com/consumer/cn/deveco-studio
-[OpenHarmony release notes]: https://gitee.com/openharmony/docs/tree/master/en/release-notes/
-[HarmonyOS NEXT commandline tools]: https://developer.huawei.com/consumer/cn/download/
-
-#### Manual installation of the HarmonyOS NEXT commandline tools
-
-The [HarmonyOS NEXT commandline tools] contain the OpenHarmony SDK and the following additional tools:
-
-- `codelinter` (linter)
-- `hstack` (crash dump stack analysis tool)
-- `hvigor` / `hvigorw` (build tool)
-- `ohpm` (package manager)
-
-Currently, the commandline tools package is not publicly available and requires a chinese Huawei account to download.
-
-#### Manual installation of hvigor without the commandline tools
```

Comment:
If you want to build on Linux without using the harmonyos commandline tools (e.g. in our CI), then this complicated section is sadly necessary. Perhaps we could move this part / some OH specific parts into a seperate file, so that we have two files? Do you think that would help make the guide easier to follow?